app/views/restaurants/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<div>
+<h1>New Restaurant Form</h1>
+
+<%= form_for [@cuisine, @restaurant] do |f| %>
+<div class="fields">
+
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+  <br/>
+
+  <%= f.label :street %>
+  <%= f.text_field :street %>
+  <br/>
+
+  <%= f.label :city %>
+  <%= f.text_field :city %>
+  <br/>
+
+  <%= f.label :state %>
+  <%= f.text_field :state %>
+  <br/>
+
+  <%= f.label :zip %>
+  <%= f.text_field :zip %>
+  <br/>
+
+  <%= f.label :phone_number %>
+  <%= f.text_field :phone_number %>
+  <br/>
+
+  <%= f.label :email %>
+  <%= f.text_field :email %>
+  <br/>
+
+  <%= f.label :website %>
+  <%= f.text_field :website %>
+  <br/>
+</div>
+  <button type="submit"><%= "Add Restaurant" %></button>
+
```

Comment:
`div` and `button` should be of the same hierarchy. Update indentation accordingly.

Reply:
done